src/test/java/hw2/ex1/Ex1Test.java

```diff
@@ -0,0 +1,118 @@
+package hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Ex1Test {
+    WebDriver driver;
+    String url = "https://jdi-testing.github.io/jdi-light/index.html";
+    String login = "Roman";
+    String pass = "Jdi1234";
+    String user = "ROMAN IOVLEV";
+    String[] header = {"HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"};
+    String[] left = {"Home", "Contact form", "Service", "Metals & Colors", "Elements packs"};
+    String[] keyWord = new String[4];
+
+    @BeforeClass
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(1, TimeUnit.SECONDS);
+        driver.manage().timeouts().pageLoadTimeout(3, TimeUnit.SECONDS);
```

Comment:
What is the reason for overriding default values?

Reply:
implicitly wait was changed to explicitly (methods for them defined in class LoginTest (package hw2).